solutions/search/ranking/semantic-reranking.md

```diff
@@ -37,11 +37,10 @@ Semantic re-ranking enables a variety of use cases:
 * **Semantic retrieval results re-ranking**
 
     * Improves results from semantic retrievers using ELSER sparse vector embeddings or dense vector embeddings by using more powerful models.
-    * Adds a refinement layer on top of hybrid retrieval with [reciprocal rank fusion (RRF)](elasticsearch://reference/elasticsearch/rest-apis/reciprocal-rank-fusion.md).
+    * Adds a refinement layer on top of hybrid retrieval with [reciprocal rank fusion (RRF)](elasticsearch://reference/elasticsearch/rest-apis/reciprocal-rank-fusion.md) or linear combination using the [linear retriever](https://www.elastic.co/docs/reference/elasticsearch/rest-apis/retrievers/linear-retriever)
 
 * **General applications**
 
-    * Supports automatic and transparent chunking, eliminating the need for pre-chunking at index time.
```

Comment:
This isn't correct, I think the original doc conflated semantic reranking with semantic text potentially.

Reply:
good call not sure where that line came from